File: Proyecto_2_Progamacion_IV/web/js/Medico.js
  function Medico(codigo,cedula,nombre,telefono,email) {
    this.Medico(codigo,cedula,nombre,telefono,email);
  }
  
  Medico.prototype={
  	codigo: -1,
        cedula: "",
        nombre: "",
        telefono: -1,
        email: "",
	Medico: function(codigo,cedula,nombre,telefono,email){
		this.codigo=codigo;
                this.cedula = cedula;
                this.nombre=nombre;
                this.telefono=telefono;
                this.email=email;
	},
	toString:function(){
	  return this.codigo+": "+this.nombre;
	}
  };
  
  Medico.from= function(plain){
    med = new Medico(plain.codigo,plain.cedula,plain.nombre,plain.telefono,plain.email);
	return med;
  };
  
    Medico.to= function(med){
    return {
        _class : 'Medico',
        codigo : med.codigo,
        cedula: med.cedula,
        nombre : med.nombre,
        telefono : med.telefono,
        email : med.email
    };	
  };

File: Proyecto_2_Progamacion_IV/src/java/proyecto/modelo/Medico.java
package proyecto.modelo;

import java.io.Serializable;

public class Medico implements Serializable, Jsonable{
    String codigo;
    int cedula;
    String nombre;
    int telefono;
    String email;

   public Medico(){
       this("",-1,"",-1,"");
   }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

     public void setNombre(String nombre) {
        this.nombre = nombre;
    }
     
    public int getTelefono() {
        return telefono;
    }

    public void setTelefono(int telefono) {
        this.telefono = telefono;
    }
    
    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

   
   public Medico(String codigo,int cedula, String nombre, int telefono, String email){
        this.codigo=codigo;
        this.cedula = cedula;
        this.nombre=nombre;
        this.telefono=telefono;
        this.email=email;
   }

    public String getNombre() {
        return nombre;
    }
}
